filesystem/README.md

```diff
@@ -359,7 +359,7 @@ Required on Android, only when using <a href="#directory">`Directory.Documents`<
 | **`path`**      | <code>string</code>                             | The path of the file to append                                                                                                              | 1.0.0 |
 | **`data`**      | <code>string</code>                             | The data to write                                                                                                                           | 1.0.0 |
 | **`directory`** | <code><a href="#directory">Directory</a></code> | The <a href="#directory">`Directory`</a> to store the file in                                                                               | 1.0.0 |
-| **`encoding`**  | <code><a href="#encoding">Encoding</a></code>   | The encoding to write the file in. If not provided, data is written as base64 encoded. Pass FilesystemEncoding.UTF8 to write data as string | 1.0.0 |
```

Comment:
This one comes from the `filesystem/src/definitions.ts` file, so should be changed there and will be automatically updated here on doc generation. If only changed here, will be undone on next doc generation.